src/webextension/list/manage/components/account-summary-label.js

```diff
@@ -2,25 +2,110 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { Localized } from "fluent-react";
 import PropTypes from "prop-types";
 import React from "react";
+import Modal from "react-modal";
+
+import { Link } from "../../../widgets/link";
 
 import styles from "./account-summary-label.css";
 
-export default function AccountSummaryLabel({displayName, avatar}) {
+function AccountSummaryLabel({displayName, avatar, onClick}) {
   if (!displayName) {
     return null;
   }
 
   return (
-    <div className={styles.accountSummary}>
-      <span className={styles.avatar}><img src={avatar} /></span>
+    <button className={styles.label} onClick={onClick}>
+      <span className={styles.avatar}><img src={avatar}/></span>
       <span className={styles.displayName}>{displayName}</span>
-    </div>
+    </button>
```

Comment:
missing chevron (see https://zpl.io/bezRwQq)

@changecourse where are the SVG assets for this?

Reply:
SVGs are added in (and I found a recipe for the little speech bubble arrow too!).